query/graphql/src/main/java/org/codice/ddf/admin/graphql/transform/FunctionDataFetcherException.java

```diff
@@ -14,40 +14,23 @@
 package org.codice.ddf.admin.graphql.transform;
 
 import java.util.List;
-import java.util.Map;
 
-import org.boon.Boon;
 import org.codice.ddf.admin.api.report.ErrorMessage;
 
-import com.google.common.collect.ImmutableMap;
-
 public class FunctionDataFetcherException extends RuntimeException {
 
-    private String functionName;
-    private Map<String, Object> args;
-
     private List<ErrorMessage> customMessage;
 
-    public FunctionDataFetcherException(String functionName, Map<String, Object> args, List<ErrorMessage> customMessage) {
-        super();
-        this.functionName = functionName;
-        this.args = args;
+    public FunctionDataFetcherException(String message, List<ErrorMessage> customMessage) {
+        super(message);
         this.customMessage = customMessage;
     }
 
     public List<ErrorMessage> getCustomMessages() {
         return customMessage;
     }
 
-    @Override
-    public String toString() {
-        return Boon.toPrettyJson(toMap());
-    }
-
-    private Map<String, Object> toMap() {
-        return ImmutableMap.of(
-                "functionName", functionName,
-                "args", args,
-                "errors", customMessage);
+    public synchronized Throwable fillInStackTrace() {
```

Comment:
There is a warning in the logs that is printed every time a GraphQL query has an error (which can happen a lot) and generated a lot of noise in the logs. I think this was to avoid logging that stack trace.